src/sentry/static/sentry/app/views/alerts/details/chart.tsx

```diff
@@ -47,24 +74,37 @@ type Props = {
 export default class Chart extends React.PureComponent<Props> {
   render() {
     const {aggregation, data, detected, closed} = this.props;
-
+    const detectedTs = detected && moment.utc(detected).unix();
+    const closedTs = closed && moment.utc(closed).unix();
     const chartData = data.map(([ts, val]) => [
       ts * 1000,
       val.length ? val.reduce((acc, {count} = {count: 0}) => acc + count, 0) : 0,
     ]);
 
-    const detectedTs = detected && moment.utc(detected).unix();
-    const closedTs = closed && moment.utc(closed).unix();
+    let detectedCoordinate: any = undefined;
```

Comment:
Let's try to type this instead of using `any` (we haven't strictly enforced this via tooling because we're transitioning), it looks like a coordinate is a tuple of timestamp and value no?

Also declaring + initializing it to `undefined` is a bit redundant here.

Reply:
Oh wait, did you not update this?

Reply:
Updated :-)